src/modules/universal-discovery/universal.discovery.module.js

```diff
@@ -194,11 +244,32 @@ export default class UniversalDiscoveryModule extends Component {
             id: TAB_SEARCH,
             panel: SearchPanelComponent
         };
+        const createPanelConfig = {
+            id: TAB_CREATE,
+            panel: CreatePanelComponent,
+            attrs: {
+                languages: this.props.languages,
+                contentTypes: this.props.contentTypes,
+                onLanguageSelected: this.onLanguageSelected,
+                onContentTypeSelected: this.onContentTypeSelected,
+                contentTypesMap: this.state.contentTypesMap,
+                forcedLanguage: this.props.cotfForcedLanguage
+            }
+        };
+
+        if (this.props.onlyContentOnTheFly) {
+            return (
+                <div className="m-ud__panels">
+                    {this.renderSinglePanel(createPanelConfig)}
```

Comment:
I'm thinking about moving these configs into some kidn of array. But maybe in other PR.